tasks/hyrax-metadata-updates/index.js

```diff
@@ -133,8 +182,9 @@ async function generatePath(config, metadata, isUmmG) {
   if (providerId === undefined) {
     throw new InvalidArgument('Provider not supplied in configuration. Unable to construct path');
   }
-  const entryTitle = await getEntryTitle(config, metadata, isUmmG);
-  return `providers/${providerId}/collections/${entryTitle}/granules/${getGranuleUr(metadata, isUmmG)}`;
+  const entryCollection = await getCollectionEntry(config, metadata, isUmmG) ;
+  //return `providers/${providerId}/collections/${entryTitle}/granules/${getGranuleUr(metadata, isUmmG)}`;
+  return `collections/${entryCollection}/granules/${getGranuleUr(metadata, isUmmG)}`;
```

Comment:
Is it intentional that this path no longer begins with `providers/${providerId}/`?

Reply:
Yes.